lib/WebRtcTransport.d.ts

```diff
@@ -6,7 +6,7 @@ export declare type WebRtcTransportOptions = {
      * Listening IP address or addresses in order of preference (first one is the
      * preferred one).
      */
-    listenIps: TransportListenIp[] | string[];
+    listenIps: Array<TransportListenIp | string>;
```

Comment:
Why this change? It was on purpose. I really prefer to use `[]` syntax rather than `Array`.